models/engine/file_storage.py

```diff
@@ -68,3 +68,26 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieves an object based on class name and id"""
+        if cls and cls in classes.values() and id and type(id) is str:
+            key = cls.__name__ + '.' + id
+            return self.__objects.get(key, None)
+        elif cls and cls in classes and id and type(id) is str:
+            key = cls + '.' + id
+            return self.__objects.get(key, None)
+        else:
+            return None
+
+    def count(self, cls=None):
+        """counts the numbers of objects in storage"""
+        if cls and cls in classes.values():
+            print("nic: class is a class name")
```

Comment:
There are some debugging lines in the code still

Reply:
got it, sorry I did use those to debug. Removing now and pushing again